### DISCUSSION TITLE
Step 8 script `Build ipa for distribution` exited with status code 65

### DISCUSSION THREAD

**Discussion by Chandra-Revscale** [codemagic-ci-cd/codemagic-docs#2660] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

React Native

### Steps to reproduce

push code for generating build on TestFlight

### Expected results

archive successful but getting error to push to TestFlight

### Actual results

Step 8 script `Build ipa for distribution` exited with status code 65

### Build id (optional)

66ab67d5eb768058c5d0adae

**Comment by dtrdic** [comment]
hey @Chandra-Revscale please try by adding the archive flag to your build command
```
              xcode-project build-ipa \
                --workspace "$CM_BUILD_DIR/ios/$XCODE_WORKSPACE" \
                --scheme "$XCODE_SCHEME" \
                --archive-flags="-destination 'generic/platform=iOS'"   
```

**Reply by Chandra-Revscale** [reply]
/var/folders/w2/rrf5p87d1bbfyphxc7jdnyvh0000gn/T/build_script_8_1_z7w0lo3k: line 6: --archive-flags=-destination 'generic/platform=iOS': No such file or directory


Build failed :|
Step 8 script `Build ipa for distribution` exited with status code 127

**Reply by dtrdic** [reply]
You are missing a backslash (`\`) after the scheme flag

**Reply by Chandra-Revscale** [reply]
added and got the code 65 error

Build failed :|
Step 8 script `Build ipa for distribution` exited with status code 65

**Reply by Chandra-Revscale** [reply]
@dtrdic any other solution for the same

**Reply by dtrdic** [reply]
Hey @Chandra-Revscale have you tried to run this project locally? The project is not compiling, the error message indicates that the module `react-native-gifted-charts/src/utils` cannot be found within the project or in the node_modules directory. 
Maybe try locally to clean your project and reinstall modules;
```
npm cache clean --force
rm -rf node_modules
npm install
```

**Comment by dtrdic** [comment]
I'm tentatively closing the discussion since we haven't heard back from you. You can open the discussion again by replying to this message or opening a new discussion.
